uiowa/uiowa
# Added custom block type layout for uiowa_program block.

Placed this here for eventual testing, not to be merged yet.

# To test
* `blt ds`
* `drush @default.local uli admin/config/development/configuration/config-split`
* Turn on the CLAS Program Block split.
* `drush @default.local cim`
* You should now be able to add the program block via LB.
